api/src/main/java/jakarta/jms/JMSContext.java

```diff
@@ -262,7 +262,9 @@
 
     /**
      * Starts (or restarts) delivery of incoming messages by the JMSContext's connection. A call to {@code start} on a
-     * connection that has already been started is ignored.
+     * connection that has already been started is ignored. Also, it is normally not necessary for application to call this
+     * method, since the the underlying connection used by the JMSContext will be started automatically when a
+     * consumer is created.
```

Comment:
```suggestion
     * connection that has already been started is ignored. Also, it is not necessary for an application to call this
     * method unless the {@code autoStart} property is set to false with {@link #setAutoStart()}, 
     * since the underlying connection used by the {@code JMSContext} will be started automatically when a consumer is created.
```